src/db/fakeData.js

```diff
@@ -0,0 +1,147 @@
+export const users = [
+    {
+        userName: "lola",
+        pass: "cyf"
+    },
+    {
+        userName: "ekip",
+        pass: "cyf"
+    },
+    {
+        userName: "denis",
+        pass: "cyf"
+    }
+]
+ export const studentProfile = [
```

Comment:
studentProfile should be just an object(not an array) with informations about a student. Also the data inside studentProfile should be organised a bit different. Each module should be an object that contains 3 key-value pairs, where the key will be each individual week and the value will be studentScore(initially 0)